### PR TITLE
Store task parent for forked tasks

### PR DESCRIPTION
**Pull request by buweixin**

### CODE REVIEW

File: openlcs/tasks/models.py [hunk 1]
```diff
@@ -43,6 +43,19 @@ class Task(models.Model):
         blank=True, null=True,
         on_delete=models.CASCADE
     )
+    # TODO(Weixin Bu): modify it later
```
Comment (qduanmu) [inline openlcs/tasks/models.py:46]: If there are changes on the model, migrations should be attached, please mark this as draft if this PR is still in progress.

Comment (buweixin) [reply openlcs/tasks/models.py:46]: OK. The migrations file is submitted here: `openlcs/tasks/migrations/0003_auto_20221118_0247.py`.
The PR is in progress.

File: openlcs/tasks/views.py [hunk 2]
```diff
@@ -73,6 +76,17 @@ def filter_traceback(self, queryset, name, value):
             traceback__icontains=value).values_list('task_id', flat=True)
         return queryset.filter(meta_id__in=meta_ids)
 
+    # def filter_child_task_ids(self, queryset, name, value):
```
[the remark this answers is not in the document]
Comment (buweixin) [reply openlcs/tasks/views.py:79]: DONE

File: openlcs/tasks/views.py [hunk 3]
```diff
@@ -38,10 +38,13 @@ class TaskFilter(filters.FilterSet):
     )
     meta_id = filters.CharFilter(field_name='meta_id', label='meta_id')
```
[the remark this answers is not in the document]
Comment (buweixin) [reply openlcs/tasks/views.py:40]: I think this line is needed to filter components with meta_id

File: openlcs/tasks/models.py [hunk 4]
```diff
@@ -43,6 +43,7 @@ class Task(models.Model):
         blank=True, null=True,
         on_delete=models.CASCADE
     )
+    parent_task_id = models.TextField(blank=True, null=True, default='')
```
[the remark this answers is not in the document]
Comment (buweixin) [reply openlcs/tasks/models.py:46]: DONE

File: openlcs/tasks/migrations/0003_remove_task_child_task_id.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('tasks', '0002_auto_20221118_0422'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='task',
+            name='child_task_id',
```
[the remark this answers is not in the document]
Comment (buweixin) [reply openlcs/tasks/migrations/0003_remove_task_child_task_id.py:15]: DONE

File: openlcs/tasks/views.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (buweixin) [reply openlcs/tasks/views.py]: DONE

**Pull request merged**